### PR TITLE
Adding a git-config prerequisite, for cloning cucumber-suite

### PR DESCRIPTION
**Pull request by MalloZup**

### CODE REVIEW

File: salt/control-node/init.sls [hunk 1]
```diff
@@ -71,9 +71,25 @@ cucumber-requisites:
 cucumber-testsuite:
   git.latest:
     - name: https://github.com/SUSE/spacewalk-testsuite-base.git
-    - rev: slenkins
+    {% if 'manager30' in grains['branch'] %}
+    - rev: manager30 
+    {% elif 'refhost30' in grains['branch'] %}
```
Comment (moio) [inline salt/control-node/init.sls]: Reference hosts are a totally different matter and should come as a separate PR.

We might even decide not to set them up with cucumber at all, so this is surely for later.

File: salt/control-node/init.sls [hunk 2]
```diff
@@ -71,9 +71,25 @@ cucumber-requisites:
...
+    - rev: manager30 
+    {% elif 'master' in grains['branch'] %}
```
Comment (moio) [inline salt/control-node/init.sls]: Please use `else` here - basically "in all other cases use master"

File: salt/control-node/init.sls [hunk 3]
```diff
@@ -71,9 +71,25 @@ cucumber-requisites:
...
+    - rev: master
+    {% endif %}
     - target: /root/spacewalk-testsuite-base
 
+git-config:
+  git.config_set:
+    - name:  http.postBuffer
+    - value: 2M
+    - global: True
+    - require:
```
Comment (moio) [inline salt/control-node/init.sls]: I think this is the other way round: `cucumber-testsuite` should require `git-config`, right?

Comment (MalloZup) [inline salt/control-node/init.sls]: yeah, true.

**Comment by MalloZup** [comment]
This can be merged only when the slenkins branch go to manager30 branch.

**Comment by MalloZup** [comment]
not needed, workaround doesn't fix.

**Pull request closed without merging**